packages/edit-post/src/components/layout/index.js

```diff
@@ -173,6 +174,7 @@ function Layout( { styles } ) {
 
 	useDrop( ref );
 	useEditorStyles( ref, styles );
+	DarkEditorStyle( ref );
```

Comment:
Using it here means it only applies to edit-post. Should we do the same in  all the other pages `edit-widgets` `edit-navigation` and `edit-site`?

Reply:
Probably edit-site, good catch. The other two, good question. @shaunandrews ?

Reply:
Both the widgets and navigation editors are more abstract representations of content, and I don't think they'll need to support this.